outbox.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<title>Distributed Press Reader</title>
+<link rel="stylesheet" href="./style.css" />
+<div id="outbox-container">
+  <!-- https://staticpub.mauve.moe/outbox.jsonld -->
+  <!-- hyper://staticpub.mauve.moe/outbox.jsonld -->
+  <distributed-outbox
+    url="ipns://staticpub.mauve.moe/outbox.jsonld"
+  ></distributed-outbox>
+</div>
+<div class="pagination-controls">
+  <button onclick="prevPage()">Previous Page</button>
```

Comment:
The scripts for these buttons should be separate from the component decleration. Ideally we should have the components be fully self contained so we can render several on a page.